### ISSUE TITLE
Podcast page - podcasts can not be started without mouse interaction

### ISSUE THREAD

**Issue by Link2Twenty** [forem/forem#14002]
**Describe the bug**
Currently, to start a podcast on the podcast page you have to click the play button, there should be a way to do this with the keyboard.

**To Reproduce**
1. Go to a [podcast](https://dev.to/devdiscuss/s5-e6-home-automation-nerd-out)
2. Press tab
3. See that the play button is skipped over

**Expected behaviour**
Play button should be in the tab index and pressing space or return should be the same as a click.

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**

The play 'button' is actually a div with an onclick set with JS.

https://github.com/forem/forem/blob/7e322e4f7b85a60ec637b38ed22387ae484642ba/app/views/podcast_episodes/show.html.erb#L52-L63

https://github.com/forem/forem/blob/c842d1a0c6718f319f167e9ec13d4d7d5d6b6d39/app/assets/javascripts/initializers/initializePodcastPlayback.js#L181-L198

**Comment by aitchiss** [comment]
Thanks for creating the issue @Link2Twenty!

Whoever picks this up might find it useful to refer to the ['Toggle buttons' section of Inclusive Components](https://inclusive-components.design/toggle-button/) which outlines some considerations for the labelling/state of play/pause buttons 🙂

**Comment by aayush-05** [comment]
I'll pick this up @cmgorton

**Comment by aitchiss** [comment]
Closed by #14139
